src/types/Asset.ts

```diff
@@ -18,6 +20,12 @@ export default interface Asset extends CreatedUpdatedProps, AbstractCurrentConsu
   values: Consumption[],
 }
 
+export enum AssetType {
+  CO = 'CO',
+  PR = 'PR',
+  CO_PR = 'CO-PR',
+}
```

Comment:
It would be nice also to have full string keys for readability?

  CONSUME = 'CO',
  PRODUCE = 'PR',
  CONSUME_PRODUCE = 'CO-PR',

As you want as I had already accepted your proposal ;-)